integrations/gitlab/gitlab_integration/ocean.py

```diff
@@ -108,7 +111,9 @@ async def on_start() -> None:
 async def resync_groups(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
     for service in get_cached_all_services():
         async for groups_batch in service.get_all_groups():
-            yield [group.asdict() for group in groups_batch]
+            tasks = [service.enrich_group_with_members(group) for group in groups_batch]
+            enriched_groups = await asyncio.gather(*tasks)
```

Comment:
when we do such things, we should add it under a feature flag, as we wouldn't want to just do all those extra requests to get the members if it wasn't actually intended by the user.

Therefor I suggest leaving the `group` kind as it was.
And add the following kinds.

`groupMembers` - which will do this exact logic that you have implemented in `resync_groups` - this kind will allow users to decide whether they want to export the group with or without members.

`member / user` - which will bring the information about a user. such as the email etc..